Hamburg/Stabi/SHOr271a.xml

```diff
@@ -1,316 +1,342 @@
+                    Probably, the pricking was made opening the quire in the middle and piercing the
+                    folios together. Indeed, the pricks are more marked in the internal bifolio than
+                    in the outer. </ab>
+                </layout>
+              </layoutDesc>
+            </objectDesc>
 
-                        <handDesc>
-                          <handNote script="Ethiopic" xml:id="h1">
-                            <seg type="ink">Black, red</seg>
-                          </handNote>
-                        </handDesc>
+            <handDesc>
+              <handNote script="Ethiopic" xml:id="h1">
+                <date notBefore="1700" notAfter="1800"/>
+                <seg type="ink">18th cent. Black, red</seg>
+                <note>According to Charles, ´It was written in the 18th cent. Is was possibly
+                  written by the same scribe as Ms.o (=Ms.orient.491 [lies: 499] in the British
+                  Museum).`</note>
+              </handNote>
+            </handDesc>
 
-                        <decoDesc>
-                            <decoNote type="punctuation" xml:id="d1">
-                              <locus target="#69vb n3"/>
-                              <desc>Two bands of black and red dots close the text.</desc>
-                            </decoNote>
-                        </decoDesc>
+            <decoDesc>
+              <decoNote type="punctuation" xml:id="d1">
+                <locus target="#69vb n3"/>
+                <desc>Two bands of black and red dots close the text.</desc>
+              </decoNote>
+            </decoDesc>
 
-                        <additions>
-                          <list>
-                          <item xml:id="a1">
-                            <desc>Donation note from J. M. Flad to his beloved brother Haussmann.</desc>
-                            <locus target="#1r"/>
-                          </item>
-                          <item xml:id="a2">
-                            <desc>Signature written in pencil:
-                            "Cod. Orient. 271a = 130 in Scrinio" </desc>
-                            <locus target="#1r"/>
-                          </item>
-                          <item xml:id="a3">
-                            <desc>Small stripe of paper: "Handschriften Sammlung
-                            Freihr. a.d. v. Westenholz Hamburg"
-                          </desc>
-                            <locus target="#1v"/>
-                          </item>
-                          <item xml:id="a4">
-                            <desc>Note written in pencil: "1905
-                            5388"</desc>
-                            <locus target="#1v"/>
-                          </item>
+            <additions>
+              <list>
+                <item xml:id="a1">
+                  <desc>Donation note from J. M. Flad to his beloved brother Haussmann.</desc>
```

Comment:
This, I suppose, is PRS4133Flad. Maybe one could add the reference?

Reply:
It is an archival record, as far as I could see, it has not been published yet. I have discussed with Denis what would be a solution and came up with the record you have seen. If you want me to refer to this archival record which needs to be treated as a manuscript, I will do it, but I would be grateful for an example.